.github/workflows/ci.yml

```diff
@@ -186,32 +180,26 @@ jobs:
         run: |
           is_ok=true
 
-          for proj in cooked-validators examples; do
-            for step in cabal-build cabal-test; do
-              echo "!! [$proj]: output from $step"
-              cat $proj-$step.out
-              res=$(cat $proj-$step.res | cut -d':' -f2)
-              if [ "$res" -ne 0 ]; then
-                is_ok=false
-              fi
-            done
+          for step in cabal-build cabal-test; do
+            echo "!! output from $step"
+            cat $step.out
+            res=$(cat $step.res | cut -d':' -f2)
+            if [ "$res" -ne 0 ]; then
```

Comment:
No. This would expand `$(cat $step.res | cut ...)` which will expand to `0` or `1`. Then the Shell will try to run the command `0` or `1` and fail (unless they exist on the system but then God know what would happen). Alternatively, we could go for:
```shell
if [ "$(cat $step.res | ...)" -ne 0 ]; then
```
but I don't find it to be an improvement.